lib/model/rwfolder.go

```diff
@@ -1345,7 +1291,12 @@ func (f *sendReceiveFolder) copierRoutine(in <-chan copyBlocksState, pullChan ch
 				state.copyDone(block)
 			}
 		}
-		weakHashFinder.Close()
```

Comment:
Yes, it doesn't need closing anymore as we pass in the io.Reader rather than the name.